test/unit/node_test.rb

```diff
@@ -9,13 +9,28 @@ class NodeTest < ActiveSupport::TestCase
     assert_equal 'page', node.type
     assert_equal 1, node.status
     assert !node.answered
+    assert_equal [], node.location_tags
   end
 
   test 'basic question attributes' do
     question = nodes(:question)
     assert question.answered
   end
 
+  test 'basic location attributes' do
+    map = nodes(:map)
+    assert_equal [], map.tags.collect(&:name)
+    assert map.has_power_tag('lat')
```

Comment:
Don't know why this is failing so I added an assertion to just dump what tagnames are actually in there... ☹️